src/EventProducerService.cpp

```diff
@@ -39,15 +39,32 @@ void EventProducerService::setProducedEvents()
 
 byte EventProducerService::createDefaultEvent(byte evValue)
 {
+  unsigned int eventNum = 0;
+  unsigned int nodeNum = module_config->nodeNum;
   byte data[4];
-  Configuration::setTwoBytes(&data[0], module_config->nodeNum);
-  data[2] = 0;
-  data[3] = evValue;
   
   byte index = module_config->findEventSpace();
```

Comment:
What if the event table is full?
No we didn't check this before either so you don't have to fix that now. Just want to raise this issue.
This will be tricky to fix as the error condition needs to be worked back to sendEvent(). Should sendEvent() ignore this?

Reply:
Good point.  I need to think about this.  It clearly needs to flag an error.